src/main/java/me/figo/FigoConnection.java

```diff
@@ -211,18 +111,13 @@ protected Gson createGson() {
      */
     public String getLoginUrl(String scope, String state) {
         try {
-            StringBuilder sb = new StringBuilder();
-            sb.append(apiEndpoint);
-            sb.append("/auth/code?response_type=code&client_id=");
-            sb.append(URLEncoder.encode(this.clientId, "ISO-8859-1"));
-            sb.append("&redirect_uri=");
-            sb.append(URLEncoder.encode(this.redirectUri, "ISO-8859-1"));
-            sb.append("&scope=");
-            sb.append(URLEncoder.encode(scope, "ISO-8859-1"));
-            sb.append("&state=");
-            sb.append(URLEncoder.encode(state, "ISO-8859-1"));
-            return sb.toString();
+            return getApiEndpoint()
```

Comment:
Yes, I saw that, but to be fully honest also was not aware of the advanced level of transformation in the Oracle Java compiler. Is this also going to be the case at least for OpenJDK and Android? If yes, something learned on my part.